src/AccessibilityInsights.Core/Enums/FileFilters.cs

```diff
@@ -14,14 +14,4 @@ public static class FileFilters
         public const string EventsExtension = ".a11yevent";
         public const string SarifExtension = ".sarif";
```

Comment:
I know we use some of these in Axe.Windows, but do we need all of the const strings? I thought we only use the sarif & test ones